docs/API.md

```diff
@@ -1,6 +1,6 @@
 # Charm store API
 
-The current live API lives at https://api.jujucharms.com/charmstore/v4
+The current live API lives at https://api.jujucharms.com/charmstore/v5
```

Comment:
I don't understand. The API is at v5 as in:

https://api.jujucharms.com/charmstore/v5/wordpress/meta/charm-metadata

right?